.github/workflows/cache-issues.yaml

```diff
@@ -10,8 +10,10 @@ jobs:
     steps:
       - run: |
           mkdir -p issues
+          mkdir -p issues/pull-requests
           for repo in scylladb scylla-enterprise scylla-manager scylla-operator scylla-cluster-tests scylla-dtest qa-tasks scylla-tools-java ; do
-            gh issue list --state all --json number,state,labels --limit 30000 --template '{{range .}}{{.number}},{{.state}},{{range .labels}}{{.name}}|{{end}}{{println ""}}{{end}}' --repo scylladb/$repo > issues/scylladb_$repo.csv
+            gh issue list --state all --json number,state,labels,title --limit 30000 --template '{{range .}}{{.number}},{{.state}},{{range .labels}}{{.name}}|{{end}},{{.title}}{{println ""}}{{end}}' --repo scylladb/$repo > issues/scylladb_$repo.csv
+            gh issue pr --state all --json number,state,labels,title --limit 30000 --template '{{range .}}{{.number}},{{.state}},{{range .labels}}{{.name}}|{{end}},{{.title}}{{println ""}}{{end}}' --repo scylladb/$repo > issues/pull-requests/scylladb_$repo.csv
```

Comment:
`--repo scylladb/$repo > issues/pull-requests/scylladb_$repo.csv` shouldn't be without issues prefix? `--repo scylladb/$repo > pull-requests/scylladb_$repo.csv `

Reply:
it's complex the rest of the things, i.e. need to do multiple stage to upload to s3